File: server/test/test/Protocols/login_res.cs
using messages;

namespace messages.Protocols
{
  public class login_res : MsgBase
  {
      public login_res()
      {
          CodeId = MsgCodeId.login_res;
      }
      public int state;    //
      public string reason;    ////原因
      public string token;    //

	}
}

File: client/butterfly/app/src/main/java/pf/com/butterfly/message/Protocols/login_res.java
package pf.com.butterfly.message.Protocols;
import android.util.Log;
import org.json.JSONArray;
import org.json.JSONObject;
import pf.com.butterfly.message.MsgBase;
import pf.com.butterfly.message.MsgCodeId;
  public class login_res extends MsgBase
  {
      public login_res()
      {
          CodeId = MsgCodeId.login_res;
      }
      public int state;    //

}
